src/components/ReleaseVersionMarkers.svelte

```diff
@@ -0,0 +1,62 @@
+<script>
+  import { getContext } from 'svelte';
+  import { Marker } from '@graph-paper/guides';
+  import { store } from '../state/store';
+  import { firefoxVersionMarkers } from '../state/product-versions';
+  import { fenixVersionMarkers } from '../state/product-versions';
+
+  export let labels = true;
+
+  const xScale = getContext('xScale');
+
+  let markers = [];
+  switch ($store.product) {
+    case 'firefox':
+      firefoxVersionMarkers.subscribe((m) => {
+        markers = m;
+      });
+      break;
+    case 'fenix':
+      fenixVersionMarkers.subscribe((m) => {
+        markers = m;
+      });
+      break;
+    default:
+      console.error(`Could not resolve version markers for ${$store.product}`);
+  }
+</script>
+
+{#if $store.product === 'firefox'}
```

Comment:
wondering if we should group this logic: `{#if ['firefox', 'fenix'].includes($store.product)` instead of having 2 identical code blocks for fenix and firefox?